com.avaloq.tools.ddk.xtext.ui/src/com/avaloq/tools/ddk/xtext/ui/templates/KeywordAwareCrossReferenceTemplateVariableResolver.xtend

```diff
@@ -0,0 +1,44 @@
+/*******************************************************************************
+ * Copyright (c) 2016 Avaloq Evolution AG and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     Avaloq Evolution AG - initial API and implementation
+ *******************************************************************************/
+
+package com.avaloq.tools.ddk.xtext.ui.templates
+
+import com.google.inject.Inject
+import com.google.inject.Singleton
+import org.eclipse.jface.text.templates.TemplateVariable
+import org.eclipse.xtext.conversion.impl.QualifiedNameValueConverter
+import org.eclipse.xtext.ui.editor.templates.CrossReferenceTemplateVariableResolver
+import org.eclipse.xtext.ui.editor.templates.XtextTemplateContext
+
+/**
+ * {@inheritDoc}
+ *
+ * Keywords are prefixed with escape characters.
+ */
+@Singleton
+public class KeywordAwareCrossReferenceTemplateVariableResolver extends CrossReferenceTemplateVariableResolver {

```

Comment:
I'd prefer Java code instead of Xtend, if Xtend features are not required: https://confluence.avaloq.net/display/ti/Xtend+Developer's+Guide

Reply:
Done. (I originally used xtend so I could use a stream implicitly.)